changelog.yaml

```diff
@@ -5,4 +5,12 @@
 #
 # (Markdown is supported and encouraged)
 
-unreleased: []
+unreleased:
+- |
+  Add support for Gradle-managed devices
+- |
+  Add support for generating baseline profiles via the `baselineprofile` plugin
+- |
+  Bump min supported AGP version to 8.1
+- |
+  Bump min supported Gradle version to 8.1
```

Comment:
nit: technically you only need the pipe char `|` if you're going for multiline strings - for single line items just doing 

```
- Add support for Gradle-managed devices
```

etc is fine. No need to change anything though, this is all temporary anyway until we ship.